ogc-custom-test/src/test/java/org/orbisgis/ogccustomtest/OGC_TestFilterEncoding.java

```diff
@@ -0,0 +1,179 @@
+package org.orbisgis.ogccustomtest;
+
+import net.opengis.fes._2_0_2.*;
+import org.junit.*;
+
+import javax.xml.bind.*;
+import java.io.*;
+
+/**
+ * Created by quillien on 05/05/2017.
+ * Test of generate class on .xsd about Filter Encoding Standard
+ */
+public class OGC_TestFilterEncoding {
+
+    Unmarshaller unmarshaller;
+    Marshaller marshaller;
+    ObjectFactory objFactory;
+
+
+    @Before
+    public void initialize() throws JAXBException{
+        unmarshaller = org.orbisgis.ogccustomtest.JaxbContainer.JAXBCONTEXT.createUnmarshaller();
+        marshaller = org.orbisgis.ogccustomtest.JaxbContainer.JAXBCONTEXT.createMarshaller();
+        marshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
+        objFactory = new ObjectFactory();
+
+    }
+
+
+
+    @Test
+    public void TestResourceIdentification() throws JAXBException {
+        InputStream xml = OGC_TestFilterEncoding.class.getResourceAsStream("filter_RessourceID.xml");
+        JAXBElement ridElement = (JAXBElement) unmarshaller.unmarshal(xml);
+        ResourceIdType rid = (ResourceIdType) ridElement.getValue();
+
+        Assert.assertEquals(ridElement.getName().getLocalPart(), "ResourceId");
```

Comment:
On your assert, add a message to make the test error more human readable by using the method : `Assert.assertEquals(String message, Object expected, Object actual)`.
It is also the same thing for all your assert.